core/tests/protractor_utils/waitFor.js

```diff
@@ -84,9 +84,20 @@ var visibilityOf = function(element, errorMessage) {
     until.visibilityOf(element), DEFAULT_WAIT_TIME_MSECS, errorMessage);
 };
 
+/**
+ * @param {Object} elementFinder - Element finding expression that
+ *                                 retrieves an element, e.g.
+ *                                 element(by.css(...))
+ */
+var elementToExist = function(elementFinder) {
```

Comment:
How is this different from visibilityOf?

Reply:
This just checks that the element exists in the DOM, not that it's visible. In this case, I agree that visibility is more what we're after, so I'll remove this and replace its usage with visibilityOf

Reply:
Done in e630f5ae2